force-app/main/default/classes/LeadTriggerHandlerTest.cls

```diff
@@ -0,0 +1,93 @@
+@IsTest
+public with sharing class LeadTriggerHandlerTest {
+@TestSetup
+static void makeData(){
+    
+}    
```

Comment:
You can remove this method if you're not actually making any data. 
Maybe you can have a method that creates a lead and you just need to send it the parameters when you call it in the test.